cmd/kubeadm/test/token_test.go

```diff
@@ -28,12 +28,11 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\:\\S{16}\n$"
 )
 
-var kubeadmPath string
+var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```

Comment:
I'm confused, are you suggesting not to have a package level flag? And if so, only use env variable to find kubeadm binary or helper `.sh`?